Source/Custom Device Support/Ballard MIL-STD-1553 Support.lvproj

```diff
@@ -563,6 +567,11 @@
 			</Item>
 			<Item Name="LV Config Read String.vi" Type="VI" URL="/&lt;resource&gt;/dialog/lvconfig.llb/LV Config Read String.vi"/>
 			<Item Name="lvanlys.dll" Type="Document" URL="/&lt;resource&gt;/lvanlys.dll"/>
+			<Item Name="Map of Messages to Value Indexes.ctl" Type="VI" URL="../../../Git/niveristand-ballard-milStd1553-custom-device/Source/Custom Device Support/Shared/Map of Messages to Value Indexes.ctl"/>
+			<Item Name="Message Definition.ctl" Type="VI" URL="../../../Git/niveristand-ballard-milStd1553-custom-device/Source/Custom Device Support/Shared/Message Definition.ctl"/>
+			<Item Name="Message Map Key.ctl" Type="VI" URL="../../../Git/niveristand-ballard-milStd1553-custom-device/Source/Custom Device Support/Shared/Message Map Key.ctl"/>
```

Comment:
These paths look really weird and are causing VIs to not correctly find dependencies.
![image](https://user-images.githubusercontent.com/29306186/142227261-e4f178e5-e06b-477b-ae55-92b535c095d6.png)

Reply:
probably wise to verify linkage through close of LV and reopen project whenever I add items to a LabVIEW project.